### ISSUE TITLE
Autodownload selection conflict

### ISSUE THREAD

**Issue by Elocai** [Bionus/imgbrd-grabber#982]
### What steps will reproduce the problem?

when save/Download images automatically ist selected, any selection in General/Download will be ignored.

### What is the expected behavior? What do you get instead?

when you select save when loading thumbnails, it doesnt work

### How often does this problem occur?

everytime

### What version of the program are you using? On what operating system?

5.4

**Comment by Bionus** [comment]
What do you mean it doesn't work?

I just tried adding some random tag to my whitelist with "Save when loading thumbnails" enabled, and all related pictures were automatically saved when I loaded a page containing some.

Here are the settings I used, maybe they're different from yours?

![image](https://user-images.githubusercontent.com/882719/27987386-2d52b6d8-640d-11e7-9297-2c615fb7b593.png)

**Comment by Elocai** [comment]
did you check under menu "save" the option "Download images automatically"?

The problem accurs only when both settings applied, took alot of trial and error to fix it by myself.

**Comment by Bionus** [comment]
While I couldn't reproduce your exact bug, I found some other issues while investigating it, caused by having both options enabled. A crash and another issue that would delete the image if we open the image window while it's not finished saving.

I can close the issue if you really can't reproduce, I'll be working on the two bugs I found in the meantime. And if the behavior you described occurs I'll try to fix it too.
